Packs/PAN-OS/ReleaseNotes/1_12_0.md

```diff
@@ -0,0 +1,21 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Added 6 commands.
+  - ***pan-os-platform-install-software***
+  - ***pan-os-platform-reboot***
+  - ***pan-os-platform-get-system-status***
+  - ***pan-os-hygiene-check-log-forwarding***
+  - ***pan-os-hygiene-check-vulnerability-profiles***
+  - ***pan-os-platform-update-ha-state***
+- Added *target* argument to the following commands
+  - ***pan-os-platform-get-arp-tables***
+  - ***pan-os-platform-get-route-summary***
+  - ***pan-os-platform-get-routes***
+  - ***pan-os-platform-get-system-info***
+  - ***pan-os-platform-get-global-counters***
+  - ***pan-os-platform-get-available-software***
+  - ***pan-os-platform-get-ha-state***
+  - ***pan-os-platform-get-jobs***
+  - ***pan-os-platform-install-software***
```

Comment:
if this is a command that you just added now (as part of the this PR's new commands), then you don't need to state it in the release-notes.

only those who you have added the **target** argument which were already implemented before (not in this PR)